spec/controllers/api/v1/breweries_controller_spec.rb

```diff
@@ -52,6 +52,10 @@
     end
 
     it "should return errors when filled out incorrectly" do
+      user = FactoryBot.create(:user)
+
+      sign_in user
+
       current_count = Brewery.count
 
```

Comment:
lines 56, 58, and 60 are all not really necessary. You're doing a block of test setup, then a break between that your `post` and then before your `returned_json` is appropriate.